packs/data/pfs-season-2-bestiary.db/cyclops-bodak.json

```diff
@@ -0,0 +1,621 @@
+                "trigger": {
+                    "value": ""
+                },
+                "weapon": {
+                    "value": ""
+                }
+            },
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.bestiary-ability-glossary-srd.sebk9XseMCRkDqRg"
+                }
+            },
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Lifesense 60 feet",
+            "sort": 300000,
+            "type": "action"
+        },
+        {
+            "_id": "HouMzEpqPvzQjVjw",
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>@Localize[PF2E.NPC.Abilities.Glossary.NegativeHealing]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "key": "ActiveEffectLike",
+                        "mode": "override",
+                        "path": "data.attributes.hp.negativeHealing",
+                        "value": true
+                    }
+                ],
+                "slug": "negative-healing",
+                "source": {
+                    "value": "Pathfinder Bestiary 2"
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                },
+                "weapon": {
+                    "value": ""
+                }
+            },
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.bestiary-ability-glossary-srd.TTCw5NusiSSkJU1x"
+                }
+            },
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Negative Healing",
+            "sort": 400000,
+            "type": "action"
+        },
+        {
+            "_id": "wfYvmKHkGU8nKveB",
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "free"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Frequency</strong> once per day;</p>\n<p><strong>Trigger</strong> The cyclops bodak is about to roll a d20;</p>\n<p><strong>Effect</strong> The bodak peers into an occluded spectrum of all possible futures. It gets a success (but not a critical success) on the roll instead of rolling.</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Frequency</strong> once per day;</p>\n<p><strong>Trigger</strong> The cyclops bodak is about to roll a d20;</p>\n<hr />\n<p><strong>Effect</strong> The bodak peers into an occluded spectrum of all possible futures. It gets a success (but not a critical success) on the roll instead of rolling.</p>"
```